client/src/components/Tags/tagService.test.js

```diff
@@ -3,7 +3,7 @@ import MockAdapter from "axios-mock-adapter";
 import { TagService } from "./tagService";
 import { createTag } from "./model";
 //import { interval } from "rxjs";
-//import { take, takeUntil } from "rxjs/operators";
+//import { take, takeUntil } from "r\xjs/operators";
```

Comment:
should this be removed?

Reply:
Yep, not sure what happened there. We should probably drop all of the commented code, I don't think any future changes will test or expand on pouchdb.